api/tests/test_application_api.py

```diff
@@ -0,0 +1,55 @@
+from typing import List
+
+import pytest
+from django.utils import timezone
+from rest_framework.reverse import reverse
+
+from applications.models import Application
+
+
+@pytest.mark.django_db
+def test_application_create(purpose, reservation_purpose, organisation, person, application_period, user_api_client):
```

Comment:
Ok, here i have a bit of a problem. My model defines user as not null, so for some reason this fails if i don't make it not null. 

Probably not clear from this merge request where the problem is, but the relevant things are

https://github.com/City-of-Helsinki/tilavarauspalvelu-core/blob/54b1df89899c49d877c87d0e137d9e12613254bc/api/applications_api.py#L75

The application serializer and viewset in there. If i set the user in the model to nullable then it works perfectly and the viewset sets the user from the request. 

Since i'm not too familiar with django rest framework, what's the proper way to handle this?